models/branches.go

```diff
@@ -44,6 +44,7 @@ type ProtectedBranch struct {
 	ApprovalsWhitelistUserIDs []int64            `xorm:"JSON TEXT"`
 	ApprovalsWhitelistTeamIDs []int64            `xorm:"JSON TEXT"`
 	RequiredApprovals         int64              `xorm:"NOT NULL DEFAULT 0"`
+	BlockOnRejectedReviews    bool               `xorm:"NOT NULL DEFAULT false"`
```

Comment:
Please add new migration for this column

Reply:
Done